src/App.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import './App.css';
-  import { Calculator } from './Components/Calculator'
-
+/* eslint-disable */
+  import Calculator from './Components/Calculator'
+/* eslint-enable */
```

Comment:
- [ ] As requested by the previous reviewer, kindly don't disable eslint. To fix the linter error you have two options.

1. Delete those 3 lines and paste this instead 👇 
```
import Calculator from './components/calculator';

```
This way the linter will pass, because `components` starts with a small letter, and the file name `calculator.js` starts with a small letter `c`
Feel free to copy and paste it but don't forget the empty line after the import statement.

2. Change the `calculator.js` file name to `Calculator.js` and delete those 3 lines and paste this instead 👇 
 ```
import Calculator from './components/Calculator';

 ```

Reply:
Hey @Sahar-AbdelSamad I'm kindly asking if its fine to call you on zoom for just less than 5 minutes

Reply:
> Hey @Sahar-AbdelSamad I'm kindly asking if its fine to call you on zoom for just less than 5 minutes

Glad I could help 👍